induction-task-do-you-know-this-is-a-test-rule/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+archivedreason:
+title: {Induction task - Do you know this is a test rule?}
+guid: {b8d2dcb2-2123-4292-aed3-5471290aedce}
+uri: {induction-task}
+created: {2022-12-06T11:04:00.000Z}
+authors: [Jimmy Chen]
+related:
+
+---
+
+## Heading from test rule - Jimmy
+
+When I create SSW Rules I need to consider the right format was used 
+
+They are including:
+
+<!--StartFragment-->
+
+1. This is the first format used
+2. This is the second format used
+3. This is the third format used
+
+<!--EndFragment-->
+
+Examples and markdown cheatsheet can be found at [Example Rule + Markdown Cheatsheet](https://ssw.com.au/rules/rule#unordered-list). This is an example rule to show you what is possible for a rule. You will see everything you need to create an awesome page.
+
+<!--endintro-->
+
+## The good example of rule
+
+Good rules should consider to include following:
+
+<!--StartFragment-->
+
+* This is the first item of an unordered list
+* This is the second item of an unordered list
+* This is the third item of an unordered list
+
+  * This is the first item of an unordered list inside another
+  * This is the second item of an unordered list inside anothe
+
+<!--EndFragment-->
+
+::: good
+![Good example](http://1.bp.blogspot.com/-lMa-GVtYHeM/U-x7N2cagBI/AAAAAAAACsQ/CK4yIriKuSI/s1600/8%2BBest%2BRules%2Bfor%2BGood%2BProgramming%2BStyle.jpg)
+:::
+
+### Videos
+The 5 Pillars of Teamwork with Ulysses Maclaren  
+`youtube: https://www.youtube.com/watch?v=DiZl22yQFKY`
+
+### Greybox
+:::greybox
+This is the test for grey box content
+:::
+
+### Feedback
+If you have any suggestions, please send to me using the template below:  
+::: email-template  
```

Comment:
Please check the Markdown for email template again - It's missing something at the end